tiled/authenticators.py

```diff
@@ -181,6 +182,19 @@ def authorization_endpoint(self) -> httpx.URL:
             cast(str, self._config_from_oidc_url.get("authorization_endpoint"))
         )
 
+    # TODO: Cache with expiration to allow for key rotation
```

Comment:
A `cachetools.TTLCache` might be appropriate here. (We use it elsewhere in the codebase, so we already have the dependency.)